app/views/admin/participants/validations/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "Participant #{t :name, scope: t_scope}" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path(@participant_profile)) %>
+
+<span class="govuk-caption-l"><%= @participant_profile.user.full_name %>
+<h1 class="govuk-heading-l"><%= t :header, scope: t_scope %></h1>
+
+<p><%= t :description, scope: t_scope %></p>
+
+<%= form_for @decision, url: {}, method: :post do |f| %>
```

Comment:
What does `url: {}` do?

Reply:
It causes the form to submit to the current path. The given hash is passed to `url_for`, which reverse_merge it with `url_options` extracted from the current route.

There's one interesting probelm I hit with it before: using `url: ''` theoretically should have the same effect, as submitting to the current path is a default for html's form tag. However - it breaks CRSF token generation as the form is given a token for path '', which is then verified agains the actual path...